src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -1093,7 +1093,7 @@ export class ResultsViewPageStore extends AnalysisStore
             this.studyIds,
             this.clinicalAttributes_profiledIn,
             this.clinicalAttributes_comparisonGroupMembership,
-            this.clinicalAttributes_customCharts,
+            this.customAttributes,
```

Comment:
adding this back fixes some issues like the results view oncoprint tab not showing custom data tab. however, now on the results view plots tab, the custom attributes are showing under clinical attribute datatype.

![Screenshot 2024-09-13 at 4 32 23 PM](https://github.com/user-attachments/assets/3aaa3db9-f6b7-42d5-b89c-d4a448915aa1)

let's fix this

Reply:
Done